internal/unit/handler.go

```diff
@@ -454,7 +496,17 @@ func (h *Handler) AddParentChild(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	handlerutil.WriteJSONResponse(w, http.StatusCreated, pc)
+	var parentPtr *uuid.UUID
```

Comment:
What does `parentPtr` mean? Please only use abbreviations that are commonly understood and easy to recognize.